lunatrace/bsl/hasura/migrations/lunatrace/1669838695525_change_cvss_tagging_table/up.sql

```diff
@@ -0,0 +1,25 @@
+DROP TABLE public.project_environmental_adjustment;
+
+
+CREATE TABLE public.project_folder_settings
+(
+    id         uuid PRIMARY KEY DEFAULT (gen_random_uuid()),
+    precedence INTEGER NOT NULL,
```

Comment:
We should add a comment that describes if lower or higher numbers are more preferred.

Reply:
done